Parse coordinates correctly

roback opened the pull request:
close #34
close #36

roback committed:
Parse coordinates correctly

close #34

dentarg commented on tests/test_parser.py:
```diff
@@ -309,3 +309,10 @@ def test_valid_links_result(self):
 
         self.assert_blog_posts_equal(posts[0], expectedPost)
+
+    def test_valid_links_result(self):
```
hehe, the name is not correct here :)

roback replied:
Fixed in cbe524fecbdc54bc839b7f3efe00309358a80fa5 :)

dentarg commented:
We talked about having longitude and latitude directly on the `Post` object, I think that makes sense.

roback committed:
Move latitude and longitude to post object

This way latitude and longitude is always set. If the post has no
coordinates, latitude and longitude will be None.

roback committed:
Use correct default datatype for tags

The tags are always set to an empty array by the parser, so this
won't break anything since it is never used.

Im not sure whether 'or ""' that is used for all strings are really
needed, but I left those in case they are.
